samcli/lib/telemetry/metric.py

```diff
@@ -349,15 +351,17 @@ def _get_execution_environment() -> str:
         """
         Returns the environment in which SAM CLI is running. Possible options are:
 
-        CLI (default) - SAM CLI was executed from terminal or a script.
-        IDEToolkit    - SAM CLI was executed by IDE Toolkit
-        CodeBuild     - SAM CLI was executed from within CodeBuild
+        CLI (default)               - SAM CLI was executed from terminal or a script.
+        other CICD platform name    - SAM CLI was executed in CICD
 
         Returns
         -------
         str
             Name of the environment where SAM CLI is executed in.
         """
+        cicd_platform = cicd.platform()
+        if cicd_platform:
+            return cicd_platform.name
```

Comment:
Where will this show up?

Reply:
where? it is in `executionEnvironment` metric